### ISSUE TITLE
Invite emails expire too fast for some use cases

### ISSUE THREAD

**Issue by Caerbannog** [metabase/metabase#14542]
**Describe the bug**
Invites expire after 48h (conclusion of #1016). This is annoying for some use case. Could the delay be extended to 1 or 2 weeks? I don't see what security such a short delay provides.

It's a UX problem from users.
Use case: if a team of a dozen people must be onboarded from time to time, it's likely that each time a few of them will be busy or unavailable for 48h. Even more likely if the invite is sent on Friday. They get a sub-par experience.

It's a UX problem for admistrators.
The administrator has to fix the situation with little feedback (only user complaints that can be vague) because there is no way to know : 
- whether invitations are still pending or accepted. From the admin console, there is no clear way to known that an invite has expired.
- that there is any other problem that could prevent users from signing-in (we've had metabase DNS blocked by company firewalls, email delivery problems, legacy browsers)

**Expected behavior**
Links in the invite email should wait 2 weeks before expiring.

**Information about your Metabase Installation:**

- Metabase version: (e.x. 0.37.7)

**Severity**
It's a paper cut, but it can be easy to fix.

**Comment by flamber** [comment]
Duplicate of #12829
